engines/ic3sa.cpp

```diff
@@ -37,6 +37,21 @@ using namespace std;
 
 namespace pono {
 
+unordered_set<PrimOp> logical_ops({ And,
+                                    Or,
+                                    // include bit-vector versions for boolector
```

Comment:
implies?

Reply:
Good point! Addressed in 6740d8fc96e23401e0dde246f9153f50f924be96